Models/FullDataSetCSV/QualificationCSV.cs

```diff
@@ -50,8 +51,9 @@ public class QualificationCSV
         public bool? OfferedInNorthernIreland { get; set; }
         [Name("Overall Grading Type")]
         public string? GradingType { get; set; }
-        [Name("Assessment Methods")]
-        public string? AssessmentMethods { get; set; }
+        [Ignore]
+        public string[]? AssessmentMethods { get; set; }
+        public string? AssessmentMethodsStringified => AssessmentMethods != null ? string.Join(", ", AssessmentMethods) : "";
```

Comment:
You still need the `[Name("Assessment Methods")]` above line 56 as otherwise the field will be named incorrectly

Reply:
sorted now and tested 

![image](https://github.com/user-attachments/assets/3ca6097a-c534-4661-b25b-1679a0adbd11)

Reply:
This output looks correct.